assets/icons/arrow-down.svg

```diff
@@ -0,0 +1,10 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
To be able to control the colors of SVG-icons you should specify the `fill={'currentColor'}` attribute for the `svg` tag and remove the same attribute from all nested tags. This will also require creating a JSX-element based on the svg-icon to embed it in the markup.
Since the svgr plugin does not work in storybook in our application, we will have to create the JSX-element manually. To do this, you can use the online service [React-SVGR](https://react-svgr.com/playground/?memo=true&ref=true&svgoConfig=%7B%0A%20%20%22plugins%22%3A%20%5B%0A%20%20%20%20%7B%0A%20%20%20%20%20%20%22name%22%3A%20%22preset-default%22%2C%0A%20%20%20%20%20%20%22params%22%3A%20%7B%0A%20%20%20%20%20%20%20%20%22overrides%22%3A%20%7B%0A%20%20%20%20%20%20%20%20%20%20%22removeTitle%22%3A%20false%2C%0A%20%20%20%20%20%20%20%20%20%20%22precision%22%3A%203%0A%20%20%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%5D%0A%7D&typescript=true). You can also explore an existing example in our project by going to `assets/icons/close.tsx`.